instructure/canvas-ios
# Feature flag course grades route

refs: MBL-13368
affects: parent
release note: none

Test plan:
- Launch Parent App
- Disable experimental features
- View a calendar event and/or an assignment with a link to course grades
  - /courses/:courseID/grades
  - narmstrong/o > CS 1400 > Week of Sept 1 > New Event!! > Grades link
- Tapping grades link should show course calendar week view

- Enable experimental features
- Tap a course
- You should be presented with new grades list
- You can't get to a page with links so there's no way to test tapping on a grades link
